doc/tutorials/access_store_ui.ipynb

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.        %store -r
</pre><p>The store magic is using the disk via the PickleShareDB. This means we can't use it for the SecretStore password. Instead, the user needs to enter it in every notebook, or we need to write our own magic that guarantees that the password is only in memory. Or, another idea would be, what happens if we add the loaded secret store into the globals().</p><p> </p><p>References:</p><p><a href="https://stackoverflow.com/questions/68427977/how-does-the-magic-store-commands-for-dataframe-work" target="_blank">https://stackoverflow.com/questions/68427977/how-does-the-magic-store-commands-for-dataframe-work</a></p><p><a href="https://github.com/ipython/ipython/blob/86d44b7f4674d5bf830afea5e6235c28a4b56bfd/IPython/core/interactiveshell.py#L615" target="_blank">https://github.com/ipython/ipython/blob/86d44b7f4674d5bf830afea5e6235c28a4b56bfd/IPython/core/interactiveshell.py#L615</a></p><p><a href="https://github.com/ipython/ipython/blob/86d44b7f4674d5bf830afea5e6235c28a4b56bfd/IPython/extensions/storemagic.py#L229C21-L229C23" target="_blank">https://github.com/ipython/ipython/blob/86d44b7f4674d5bf830afea5e6235c28a4b56bfd/IPython/extensions/storemagic.py#L229C21-L229C23</a></p><p><a href="https://github.com/ipython/ipython/blob/86d44b7f4674d5bf830afea5e6235c28a4b56bfd/IPython/core/interactiveshell.py#L615" target="_blank">https://github.com/ipython/ipython/blob/86d44b7f4674d5bf830afea5e6235c28a4b56bfd/IPython</a></p><p><a href="https://github.com/ipython/ipython/blob/86d44b7f4674d5bf830afea5e6235c28a4b56bfd/IPython/core/interactiveshell.py#L615" target="_blank">/core/interactiveshell.py#L615</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Agree. Will change</p>